core/templates/dev/head/services/StateStatsServiceSpec.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for state stats service.
+ */
+
+// TODO(brianrodri).
```

Comment:
Just a note that this should be done in this PR (I think you're planning to do that but thought I'd mention it so that the review comments constitute a complete list of TODOs).